tests/unit_tests/ensemble_evaluator/test_snapshot.py

```diff
@@ -138,15 +136,16 @@ def test_update_forward_models_in_partial_from_multiple_cloudevents(snapshot):
             }
         )
     )
-    forward_models = partial.to_dict()["reals"]["0"]["forward_models"]
+    forward_models = snapshot.to_dict()["reals"]["0"]["forward_models"]
     assert forward_models["0"]["status"] == state.FORWARD_MODEL_STATE_FAILURE
     assert forward_models["1"]["status"] == state.FORWARD_MODEL_STATE_FINISHED
 
 
-def test_that_realization_success_message_updates_state(snapshot):
+def test_that_realization_success_message_updates_state(
+    snapshot,
+):  # JONAK: Why use snapshot fixture here?
```

Comment:
yeah, why?? :)

Reply:
My suspicions were correct, it was not needed :)